app/controllers/nonsipp/memberdetails/UploadMemberDetailsController.scala

```diff
@@ -69,6 +72,8 @@ class UploadMemberDetailsController @Inject()(
     for {
       initiateResponse <- uploadService.initiateUpscan(callBackUrl, successRedirectUrl, failureRedirectUrl)
```

Comment:
FYI, I'm changing the evaluation of the callback url here as a part of another ticket, in case there are conflicts.